### ISSUE TITLE
A bit sad

### ISSUE THREAD

**Issue by ChrisHoffmann96** [lhunath/guide.bash.academy#67]
Hey awesome people,
i really love the way that site teaches you Bash, and i would love to recommend it to my fellow students, but it seems that the project wont be finished. I just wanted to know if you are abandoning this project or if there are plans to complete the guide? I would love to help, but i dont know much about bash (thats how i found it, i searched for a good guide and i was struck because of how good the first chapters are). Nevertheless, I'm part of the students council at my University so i would definitely make it more public & recommend it once its finished

Cheers

**Comment by McDuckQuack** [comment]
I absolutely agree! The quality of this guide is just staggering and I really enjoyed learning with it. If there is any way, I would also love to help you, although there is nothing that I know about Bash, that I didn't learn from your site. I already recommended it to some people and if it will be finished, I would spread it like a disease.

See ya!

**Comment by ptaczek** [comment]
Hi there, I was about asking teh same. I haven't found better bash resources. Similar to The Book of Shaders which is also unfinished.

@lhunath was it the FBI or KGB ?

**Comment by lhunath** [comment]
Hey folks,

First of all, thanks so much for the praise. I set out to speak with a voice that was different from the existing either very technical or rather chaotic information already out there. I'm glad to hear I succeeded and that it's been so helpful!

In 2017, I began working with Manning to publish a book on Bash based on the guide and inspired by its tone. At this point, my efforts were transferred from writing the online guide to writing the manuscript. Alas, life caught up with me and the book was left unfinished. My work on Master Password, now [Spectre](https://spectre.app), took too much of my time. Would love to clone myself and do multiple things, but unfortunately I need to pick and choose, and at this point in time, my time priority goes to [Spectre](https://spectre.app).

I can only hope that the guide has been useful to get your toes wet enough to want to dive more into the subject matter. The best resource is still the excellent [Bash man page](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html). Second to that I can recommend [Greg Wooledge's wiki](https://mywiki.wooledge.org/) and [my original guide there](https://mywiki.wooledge.org/BashGuide), which is more complete. Maybe when things calm down a bit I can come back to my work-in-progress and finish the book, but no promises. :–)

**Comment by ptaczek** [comment]
Yes, I looked at the commit history and it became clear where the focus is and I understand. And you're absolutely right, the information out there is either too technical or chaotic. So thank you for your shot, it was a good one ;) Bookmarking your two links. exit 0

**Comment by xmready** [comment]
Would much rather have this finished than a book. This is a much better interface for learning than a physical manual.
